packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -94,6 +75,32 @@ export function activate(context: vscode.ExtensionContext) {
     debugConfigProvider,
     checkpointsView
   );
+
+  console.log('in activate, getting line breakpoint info');
```

Comment:
@JimSuplizio - Can we remove this `console.log`?

CC: @DatGuyJonathan , @lcampos